src/main/java/org/folio/des/scheduling/acquisition/EdifactScheduledJobInitializer.java

```diff
@@ -23,24 +26,31 @@ public class EdifactScheduledJobInitializer {
   private final FolioExecutionContextHelper contextHelper;
   private final AcqSchedulingProperties acqSchedulingProperties;
   private final ExportJobScheduler exportJobScheduler;
+  private final boolean isQuartzEdifactEnabled;
+  //mod-data-export-spring version in which quartz scheduling for edifact order export was enabled
+  private final SemVer quartzEdifactMinVersion = new SemVer("3.0.0-SNAPSHOT");
 
-  public void initAllScheduledJob() {
-    log.info("Initialize EDIFACT scheduled job: is module registered: {} ", contextHelper.isModuleRegistered());
+  public void initAllScheduledJob(TenantAttributes tenantAttributes) {
+    log.info("initAllScheduledJob:: initialize EDIFACT scheduled job: is module registered: {}, " +
+        "tenantAttributes: {}", contextHelper.isModuleRegistered(), tenantAttributes);
     List<ExportConfig> exportConfigs = new ArrayList<>();
     try {
-      boolean isJobScheduleAllowed = isJobScheduleAllowed(acqSchedulingProperties.isRunOnlyIfModuleRegistered(),
-                                                          contextHelper.isModuleRegistered());
-      if (isJobScheduleAllowed) {
+      boolean shouldScheduleInitialConfigs = (isQuartzEdifactEnabled && shouldMigrateSchedulesToQuartz(tenantAttributes,
+        quartzEdifactMinVersion)) || (!isQuartzEdifactEnabled && isJobScheduleAllowed(
+        acqSchedulingProperties.isRunOnlyIfModuleRegistered(), contextHelper.isModuleRegistered()));
+      if (shouldScheduleInitialConfigs) {
         ExportConfigCollection exportConfigCol = basedConfigManager.getConfigCollection(ALL_EDIFACT_ORDERS_CONFIG_QUERY, Integer.MAX_VALUE);
         exportConfigs = exportConfigCol.getConfigs();
         for (ExportConfig exportConfig : exportConfigs) {
           List<Job> scheduledJobs = exportJobScheduler.scheduleExportJob(exportConfig);
           scheduledJobs.forEach(scheduledJob -> log.info("InitialJob scheduled: {}", scheduledJob.getId()));
         }
+        log.info("initAllScheduledJob:: scheduling configurations loaded and jobs scheduled successfully");
       }
     }
-    catch (Exception exception) {
-      log.error("Exception for initial EDIFACT scheduling : " + exportConfigs.size());
+    catch (Exception ex) {
+      log.error("initAllScheduledJob:: Exception for initial EDIFACT scheduling with configs size: {}",
```

Comment:
Have some doubts about such error handling. For inmemory scheduling loading initial jobs from saved configurations did not cause exception on module update(tenant install). So for quartz did not change this as well. But probably there should be some other handling?

Reply:
will add a point about checking logs to readme.